exec.go

```diff
@@ -21,19 +19,19 @@ var execCommand = cli.Command{
 			Value: "",
 			Usage: "specify the pty slave path for use with the container",
 		},
+		cli.BoolFlag{
+			Name:  "tty, t",
+			Usage: "allocate a pseudo-TTY",
+		},
 	},
 	Action: func(context *cli.Context) {
-		config, err := loadProcessConfig(context.Args().First())
```

Comment:
Will probably need a path where there is an attempt to load a process config file for the exec.  The reason for this is the way start uses the config.json file..  While there is no definitive requirement yet for the process config as a separate file in the opencontainers/specs, there is a high likelihood that this pattern in start of being able to point to a file name to say what the options were when the command was executed, will need to still be supported for high level containerd like users of runc.